src/mlj_interface.jl

```diff
@@ -67,6 +43,75 @@ MMI.metadata_model(
     reports_feature_importances=false
 )
 
+"""
+$(MMI.doc_header(MaxnetBinaryClassifier))
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X, y)
+
+where
+
+- `X`: any table of input features (eg, a `DataFrame`) whose columns
+  each have one of the following element scitypes: `Continuous` or `<:Multiclass`.
+
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `<:Binary`. The first class should refer to background values,
+  and the second class to presence values.
+
+# Hyper-parameters
+
+- `features`: Specifies which features classes to use in the model, e.g. "lqh" for linear, quadratic and hinge features. 
+    See also [Maxnet.maxnet](@ref)
+- `regularization_multiplier = 1.0`: 'Adjust how tight the model will fit. Increasing this will reduce overfitting.
+- `regularization_function`: A function to compute the regularization of each feature class. Defaults to `Maxnet.default_regularization`
+- `addsamplestobackground = true`: Controls wether to add presence values to the background.
+- `n_knots = 50`: The number of knots used for Threshold and Hinge features. A higher number gives more flexibility for these features.
+- `weight_factor = 100.0`: A `Float64` value to adjust the weight of the background samples.
+- `link = CloglogLink()`: The link function to use when predicting. See `Maxnet.predict` 
```

Comment:
Reference to functions not owned by MLJ should be qualified. If this is `Maxnet` then make this change:

```suggestion
- `link = Maxnet.CloglogLink()`: The link function to use when predicting. See `Maxnet.predict` 
```

Reply:
It's re-exported from `GLM` but I guess it's then better to refer to it as Maxnet?

Reply:
Maybe `GLM` is better because that's the package with the detailed documentation. But happy for you decided.